### PR TITLE
Small fixes and improvements

### PR DESCRIPTION
**Pull request by cookiedan42**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -561,12 +561,6 @@ Step 4. Since the value of `Email` (compulsory field) for the second contact is
 
 Step 3. CONNECTIONS will not display the first contact's invalid `Tag` and will not display the second contact. The other fields and contacts will be displayed as per normal.  
 
-
```
Comment (cookiedan42) [inline docs/DeveloperGuide.md:564]: This block belonged to mailingList and was left behind at the bottom here

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1336,8 +1330,14 @@ testers are expected to do more *exploratory* testing.
 
     1. Prerequisites: Filter contacts using the `find` command.
 
-    1. Test case: `mailingList` as above<br>
-        Expected: Same result as exporting full contact list, but with only the contacts in the filtered list
+    1. Test case: variations on `mailingList` as above<br>
+        Expected: Same result as exporting full contact list, but with only the contacts displayed in the filtered list
+
+1. Export Empty contact list
+        1. Prerequisites: Filter contacts using the `find` command.
```
Comment (yucheng11122017) [inline docs/DeveloperGuide.md:1337]: Might want to adjust the spacing here

**Review by yucheng11122017 — APPROVED** [review]
LGTM except for a minor detail

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Jiwei99) [reply docs/DeveloperGuide.md:1337]: yeah if not there will be inconsistencies in numbering.

**Review by Jiwei99 — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1336,8 +1330,14 @@ testers are expected to do more *exploratory* testing.
```
Comment (Justinhoejj) [inline docs/DeveloperGuide.md:1330]: Extra space

**Pull request merged**